codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/auth/SigV4AsymmetricAuthSchemeIntegration.kt

```diff
@@ -66,7 +66,7 @@ private class SigV4AsymmetricAuthSchemeHandler : AuthSchemeHandler {
 
     override fun instantiateAuthSchemeExpr(ctx: ProtocolGenerator.GenerationContext, writer: KotlinWriter) {
         val signingService = AwsSignatureVersion4Asymmetric.signingServiceName(ctx.service)
-        writer.write("#T(#T, #S)", RuntimeTypes.Auth.HttpAuthAws.SigV4AsymmetricAuthScheme, RuntimeTypes.Auth.Signing.AwsSigningStandard.DefaultAwsSigner, signingService)
+        writer.write("#T(#T, #S, config.sigv4aSigningRegionSet)", RuntimeTypes.Auth.HttpAuthAws.SigV4AsymmetricAuthScheme, RuntimeTypes.Auth.Signing.AwsSigningStandard.DefaultAwsSigner, signingService)
```

Comment:
**Correctness**: We're referencing a client config property here that doesn't exist anywhere in **smithy-kotlin**. I think the addition of that config property should be moved from **aws-sdk-kotlin** into this repo, possibly into this integration.